conf/homeserver.yaml

```diff
@@ -1272,20 +1272,20 @@ enable_registration: __ALLOWED_ACCESS__
 # The user must provide all of the below types of 3PID when registering.
 #
 registrations_require_3pid:
- - email
+ - __REGISTRATIONS_REQUIRE_3PID__
```

Comment:
Well for me it's not really clear how you manage this value. For me it's a list of value so the user should be able to chose `nothing`, `email` (only), `msisdn` or `email` and `msisdn` (both). And the default value should be the previous value so `email` only.

Reply:
here, @thardev could contribute a `getter/setter` with some `sed` like here https://github.com/YunoHost-Apps/mautrix_whatsapp_ynh/blob/expose-config-panel/scripts/config#L74-L81

Maybe we have to comment out `#registrations_require_3pid:` when no 3pid is required

Reply:
solved thanks to @rosbeef